package.json

```diff
@@ -53,5 +55,8 @@
   "lint-staged": {
     "*.{ts,tsx}": "eslint --cache --fix",
     "*.{js,css,md,ts,tsx,yml}": "prettier --write"
+  },
+  "dependencies": {
+    "playwright": "^1.44.1"
```

Comment:
Requesting that we move `playwright` to `devDependencies`.

Reply:
@JakeLo123 agreed. We would not want this as a regular dependency as it will cause consuming applications to install it.